projects/TPVFormer/tpvformer/nuscenes_dataset.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
suggest to support NuScenesSegDataset in another PR.

Reply:
Later we will merge the tpvformer into the package and share the `NusSegDataset` with the other methods of the lidar seg